taxi_service/taxi/admin.py

```diff
@@ -0,0 +1,34 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+
+from taxi_service.taxi.models import Manufacturer, Car, Driver
+
+
+@admin.register(Manufacturer)
+class ManufacturerAdmin(admin.ModelAdmin):
+    pass
+
+
+@admin.register(Car)
+class CarAdmin(admin.ModelAdmin):
+    search_fields = ["model"]
+    list_filter = ["manufacturer"]
+
+
+@admin.register(Driver)
+class DriverAdmin(admin.ModelAdmin):
+    list_display = [
```

Comment:
Everything is fine! GJ!
maybe just add license number to UserAdmin.list_display?

Reply:
All must be registered

Reply:
maybe for driver be better to inherit from UserAdmin?

Reply:
It is in models, in admin.py it is admin.ModelAdmin if i'm not incorrect